passes/sat/clk2fflogic.cc

```diff
@@ -124,7 +127,7 @@ struct Clk2fflogicPass : public Pass {
 								wport, log_id(module), log_id(cell), log_signal(clk),
 								log_signal(addr), log_signal(data));
 
-						Wire *past_clk = module->addWire(NEW_ID);
+						Wire *past_clk = module->addWire(stringf("\\%s$%d#past_clk#%s", log_id(cell), wport, log_signal(clk)));
```

Comment:
This doesn't check if such an identifier already exists. Also, except for very few exceptions, the `\\...` names should only be used for identifiers taken from the original HDL we are processing. Auto-generated names should always be of the `$...` kind.